### PR TITLE
delete(TransferFlowcell class)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

As Part of #2097 (deprecation of cgstats) this PR removes the `TransferFlowCell` class. The transferflowcell class used to integrate adding flow cell data to cgstats, statusdb and housekeeper.

the `cg transfer flow-cell` cmd is now removed 

The functionality of the transfer flow cell class is taken over by the `DemuxPostProcessingAPI`.
This functionality includes:
- Adding a flow cell to statusdb
- Updating sample reads and sequenced at dates
- Adding samples to a flow cell
- Parsing relevant sequencing metrics into statusDB

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by islean — APPROVED** [review]
If it is not used, this is beautiful!

File: cg/meta/transfer/__init__.py [hunk 1]
```diff
@@ -1,2 +1 @@
-from .flowcell import TransferFlowCell
 from .lims import PoolState, SampleState, TransferLims
```
Comment (henrikstranneheim) [inline cg/meta/transfer/__init__.py:1]: Out of scope - We should fix this import: it is relative and also implicit

**Review by henrikstranneheim — APPROVED** [review]
Beautiful 👍

File: cg/meta/transfer/__init__.py (hunk 1 above)
Comment (ChrOertlin) [reply cg/meta/transfer/__init__.py:1]: fixed it!

**Comment by ChrOertlin** [comment]
stage: repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/production/flow_cells/230907_A00621_0903_AHJT27DRX3
done.
Log deploy... done.
cg, version 47.0.0
prod:
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/production/flow_cells/230907_A00621_0903_AHJT27DRX3
done.
Log deploy... done.
cg, version 47.0.1

**Pull request merged**